mne/io/cnt/tests/test_cnt.py

```diff
@@ -26,7 +26,8 @@ def test_data():
     with warnings.catch_warnings(record=True) as w:
         raw = _test_raw_reader(read_raw_cnt, montage=None, input_fname=fname,
                                eog='auto', misc=['NA1', 'LEFT_EAR'])
-    assert_true(all('meas date' in str(ww.message) for ww in w))
+    assert_true(all('meas date' in str(ww.message) or
+                    'number of bytes' in str(ww.message) for ww in w))
```

Comment:
Auto mode works. Setting ``n_bytes=4`` doesn't actually break it because there is no information of the length of the data.